components/eam/src/chemistry/mozart/lin_strat_chem.F90

```diff
@@ -369,7 +383,8 @@ subroutine linv3_strat_chem_solve(ncol, lchnk, xvmr, h2ovmr, xsfc, o3col,temp, s
        xsfc(2,:)=   linoz_n2o_clim(:,pver) !  n2o (constant throughout latitude)
        xsfc(3,:)=   linoz_o3lbs(:,pver)*3.e-3_r8  !noylnz
        xsfc(4,:)=   linoz_ch4_clim(:,pver) ! ch4 (constant throughout latitude)
-       ch4max =     maxval(linoz_ch4_clim(1:ncol,pver)) 
+
+       ch4max =     maxval(linoz_ch4_avg_srf(1:ncol,:)) 
```

Comment:
linoz_ch4_avg_srf should be a single number for each timestep.

We should also abandon the miss-leading "ch4max" variable. What about renaming it to "ch4avg"?

Reply:
So we don;t need maxval.
then what about 
ch4avg=linoz_ch4_avg_srf
?

Reply:
That should be good.